.github/workflows/sql-workbench-release-workflow.yml

```diff
@@ -46,13 +46,13 @@ jobs:
         with:
           timeout_minutes: 60
           max_attempts: 3
-          command: cd kibana/plugins/sql-workbench; yarn kbn bootstrap
+          command: cd kibana/plugins/workbench; yarn kbn bootstrap
```

Comment:
line 42 `mv sql-workbench kibana/plugins` should be `mv workbench kibana/plugins`

Reply:
Done, thanks!